backend/dataall/modules/worksheets/services/worksheet_service.py

```diff
@@ -122,3 +129,55 @@ def run_sql_query(session, uri, worksheetUri, sqlQuery):
         )
 
         return AthenaClient.convert_query_output(cursor)
+
```

Comment:
Why not adding `@TenantPolicyService.has_tenant_permission(MANAGE_WORKSHEETS)` to run_nlq?

Reply:
My understanding of `MANAGE_...` tenant level permissions is that they should be for mutating operations (i.e. creates, updates, deletes) on data.all but not for others

I think this has more to do with this open PR on re-working some of our Tenant Perms https://github.com/data-dot-all/dataall/pull/1630

let me know your thoughts - curious to think if you disagree

Reply:
Makes sense. Plus, the action is already protected by a resource permission which is more restrictive than a tenant permission